.github/workflows/traiage.yaml

```
@@ -0,0 +1,151 @@
+name: AI Triage Automation
+
+on:
+  workflow_dispatch:
+    inputs:
+      issue_url:
+        description: 'GitHub Issue URL to process'
+        required: true
+        type: string
+      template_name:
+        description: 'Coder template to use for workspace'
+        required: true
+        default: 'ai-workspace'
+        type: string
+      prefix:
+        description: 'Prefix for workspace name'
+        required: false
+        default: 'traiage'
+        type: string
+      persistence_mode:
+        description: 'Persistence mode (push or archive)'
+        required: false
+        type: choice
+        options:
+          - push
+          - archive
+        default: 'archive'
+
+jobs:
+  traiage:
+    name: Triage GitHub Issue with Claude Code
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    env:
+      CODER_URL: ${{ secrets.TRAIAGE_CODER_URL }}
+      CODER_SESSION_TOKEN: ${{ secrets.TRAIAGE_CODER_SESSION_TOKEN }}
+      TEMPLATE_NAME: ${{ inputs.template_name }}
+    permissions:
+      contents: read
+      issues: write
+      actions: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+
+      - name: Extract context key from issue
+        id: extract-context
+        env:
+          ISSUE_URL: ${{ inputs.issue_url }}
+        run: |
+          issue_number="$(gh issue view "${ISSUE_URL}" --json number --jq '.number')"
+          context_key="gh-${issue_number}"
+          echo "context_key=${context_key}" >> "${GITHUB_OUTPUT}"
+          echo "CONTEXT_KEY=${context_key}" >> "${GITHUB_ENV}"
+
+      - name: Download and install Coder binary
+        shell: bash
+        env:
+          CODER_URL: ${{ secrets.TRAIAGE_CODER_URL }}
+        run: |
+          if [ "${{ runner.arch }}" == "ARM64" ]; then
+            ARCH="arm64"
+          else
+            ARCH="amd64"
+          fi
+          mkdir -p "${HOME}/.local/bin"
+          curl -fsSL --compressed "$CODER_URL/bin/coder-linux-${ARCH}" -o "${HOME}/.local/bin/coder"
+          chmod +x "${HOME}/.local/bin/coder"
+          echo "$HOME/.local/bin" >> "${GITHUB_PATH}"
+          "$HOME"/.local/bin/coder version
+          "$HOME"/.local/bin/coder whoami
```

```suggestion
          export PATH="${PATH}:${HOME}/.local/bin"
          coder version
          coder whoami
```
Minor suggestion to simplify future changes.